healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceUpdate.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package snippets.healthcare.fhir.resources;
+
+// [START healthcare_update_resource]
+
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.healthcare.v1.CloudHealthcare;
+import com.google.api.services.healthcare.v1.CloudHealthcareScopes;
+import com.google.auth.http.HttpCredentialsAdapter;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+
+public class FhirResourceUpdate {
+  private static final String FHIR_NAME =
+      "projects/%s/locations/%s/datasets/%s/fhirStores/%s/fhir/%s/%s";
+  private static final JsonFactory JSON_FACTORY = new JacksonFactory();
+  private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+
+  public static void fhirResourceUpdate(String resourceName, String data)
+      throws IOException, URISyntaxException {
+    // String resourceName =
+    //    String.format(
+    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "resource-type",
+    // "resource-id");
+    // The following data works with a Patient resource and is not intended to work with
+    // other types of FHIR resources. If necessary, supply new values for data that
+    // correspond to the FHIR resource you are patching.
+    // String data = "[{\"op\": \"replace\", \"path\": \"/active\", \"value\": false}]";
+
+    // Initialize the client, which will be used to interact with the service.
+    CloudHealthcare client = createClient();
+
+    HttpClient httpClient = HttpClients.createDefault();
+    String uri = String.format("%sv1/%s", client.getRootUrl(), resourceName);
+    URIBuilder uriBuilder = new URIBuilder(uri).setParameter("access_token", getAccessToken());
```

Comment:
I hate this, but can't think of a decent suggestion on fixing.  Not sure why you hardcode v1 in this instance either, well, I can guess.

Reply:
Copied from FhirResourcePatch lol. I can make improvements though.